src/main/java/org/folio/linked/data/configuration/kafka/KafkaListenerConfiguration.java

```diff
@@ -37,16 +37,16 @@ public FolioKafkaProperties folioKafkaProperties() {
   }
 
   @Bean
-  public ConcurrentKafkaListenerContainerFactory<String, DataImportEvent> dataImportListenerContainerFactory() {
-    var factory = new ConcurrentKafkaListenerContainerFactory<String, DataImportEvent>();
+  public ConcurrentKafkaListenerContainerFactory<String, SourceRecordDomainEvent> srsEventListenerContainerFactory() {
+    var factory = new ConcurrentKafkaListenerContainerFactory<String, SourceRecordDomainEvent>();
     factory.setBatchListener(true);
-    factory.setConsumerFactory(dataImportEventConsumerFactory());
+    factory.setConsumerFactory(sourceRecordDomainEventConsumerFactory());
```

Comment:
Shouldn't we autowire sourceRecordDomainEventConsumerFactory bean here as method param?

Reply:
makes sense, done